### ISSUE TITLE
Error on project-config/apply with 1.0.0-beta.18

### ISSUE THREAD

**Issue by billmn** [putyourlightson/craft-sprig#42]
I have installed the 1.0.0-beta.18 and now the command `project-config/apply` returns this error:

```
Your project config files were created for different versions of Craft and/or plugins than what’s currently installed.

Sprig is installed with schema version of 1.0.0-beta.18 while 1.0.0 was expected.

Try running `composer install` from your terminal to resolve.
```

I have already tried to rebuild the config file but the problem is still there.

### Versions
- Plugin version: 1.0.0-beta.18
- Craft version:  3.5.13.2

**Comment by bencroker** [comment]
Did you update from a previous beta version of Sprig?

**Comment by billmn** [comment]
Yes, from 1.0.0-beta.17

**Comment by bencroker** [comment]
Can you test with the fix in https://github.com/putyourlightson/craft-sprig/commit/7f1bbcc023df0f6b18f6cdef3533300c6e1b86d6?

```
composer require putyourlightson/craft-sprig:dev-develop
```

**Comment by billmn** [comment]
Ok, now it works. Thank you 👍

**Comment by bencroker** [comment]
Great, released in 1.0.0-beta.18.1.
